src/main/java/io/percy/selenium/Percy.java

```diff
@@ -96,65 +120,91 @@ public void snapshot(String name, List<Integer> widths) {
      * @param widths    The browser widths at which you want to take the snapshot.
      *                  In pixels.
      * @param minHeight The minimum height of the resulting snapshot. In pixels.
+     * @param enableJavaScript Enable JavaScript in the Percy rendering environment
      */
-    public void snapshot(String name, @Nullable List<Integer> widths, Integer minHeight) {
+    public void snapshot(String name, @Nullable List<Integer> widths, Integer minHeight, boolean enableJavaScript) {
```

Comment:
Worth noting that you're also adding `enableJavaScript` support here :+1:

Reply:
(in hindsight I see that in the PR description now)

Reply:
Yeah, I remember back to the support convo I had with someone that used Web Components and the only way we support that at the moment is by enabling javascript (which they couldn't do in the Java SDK 😢 )